scripts/extension-version.js

```diff
@@ -34,7 +34,7 @@ function nextExtensionVersion({
 }
 
 function stripPreReleaseText(version) {
-  return version.replace('-alpha', '').replace('-beta', '').replace('-dev', '')
+  return version.replace('-alpha', '').replace('-beta', '').replace('-insider', '')
```

Comment:
I thinks this is wrong and was just prep for us switching from `-alpha` to `-dev` in the naming of the Npm versions.